Gruntfile.js

```diff
@@ -138,15 +138,25 @@ module.exports = function(grunt) {
     },
     eslint: {
       src: ['*.js', 'lib/**/*.js', 'test/**/*.js']
+    },
+    mochaTest: {
+      test: {
+        options: {
+          run: true
+        },
+        src: ['lib/*spec.js']
```

Comment:
:+1: the `**` is for globbing any amount of sub-directories, we already have subdirectories and would ideally be adding unit tests to those in the future.